controllers/nodehealthcheck_controller_test.go

```diff
@@ -1038,6 +1039,40 @@ var _ = Describe("Node Health Check CR", func() {
 		})
 
 		Context("control plane nodes", func() {
+
+			var pdb *policyv1.PodDisruptionBudget
+			BeforeEach(func() {
+				// create etcd namespace
+				ns := &v1.Namespace{
+					ObjectMeta: metav1.ObjectMeta{
+						Name: "openshift-etcd",
+					},
+				}
+				// namespaces can't be deleted!
+				err := k8sClient.Create(context.Background(), ns)
+				if err != nil {
+					Expect(errors.IsAlreadyExists(err)).To(BeTrue())
```

Comment:
I didn't think at this solution :smile:

Reply:
yeah, looks a bit strange, but does the job 😎